compose/foundation/foundation/src/uikitTest/kotlin/androidx.compose.foundation/text/selection/TextFieldSelectionManagerTest.uikit.kt

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.text.selection
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Rect
+import kotlin.experimental.ExperimentalNativeApi
+import kotlin.test.Test
+
+@OptIn(ExperimentalNativeApi::class)
+class TextFieldSelectionManagerTest {
+    @Test
+    fun isSelectionHandleIsVisible_ltr_text() {
+        // Text = "I am TextField"
+        val isStartVisible = isSelectionHandleIsVisible(
+            true,
+            Offset(94.58f, 50.0f),
+            50.0f,
+            Rect(Offset(0f, 0f), Offset(744.0f, 50.0f))
+        )
+        val isEndVisible = isSelectionHandleIsVisible(
+            false,
+            Offset(272.38f, 50.0f),
+            50.0f,
+            Rect(Offset(0f, 0f), Offset(744.0f, 50.0f))
+        )
+        assert(isStartVisible && isEndVisible)
```

Comment:
- It's better to use `assertTrue` or `assertFalse` to verify visible and invisible results.
- please use one assert for every condition, like:
```
assertTrue(isStartVisible)
assertFalse(isEndVisible)
```